manage.py

```diff
@@ -3,7 +3,7 @@
 
 if __name__ == "__main__":
     app = create_app()
-    server = Server(host="0.0.0.0", port=5000)
+    server = Server(host="0.0.0.0", port=5000, use_debugger=True)
```

Comment:
보안상 user_debugger 는 deault 값이 false 이므로 별도로 True 로 설정하지않는게 좋을 것 같습니다.
필요하다면 커맨드라인에 -d 붙여서 사용 부탁 드립니다.
해당 커멘트 어드레스 되면 머지할께요
https://flask-script.readthedocs.io/en/latest/
```
use_debugger – Flag whether to default to using the Werkzeug debugger.
This can be overriden in the command line by passing the -d or -D flag.
Defaults to False, for security.
```

Reply:
알겠습니다